src/Pact/Repl/Lib.hs

```diff
@@ -144,6 +146,14 @@ replDefs = ("Repl",
      "Output VALUE to terminal as unquoted, unescaped text."
      ,defZRNative "env-hash" envHash (funType tTyString [("hash",tTyString)])
      "Set current transaction hash. HASH must be a valid BLAKE2b 512-bit hash. `(env-hash (hash \"hello\"))`"
+     ,defZNative "test-capability" testCapability
+      (funType tvA [("capability", TyFun $ funType' tTyBool [])]) $
+     "Allow the testing of the scope of a capability CAPABILITY in the repl environment. This is intended for testing " <>
+     "ONLY, and one should revoke all capabilities after by calling 'revoke-all-capabilities' after use." <>
+     "`$(grant-capability (TRANSFER) (my-module.transfer sender receiver 1.0))`"
+     ,defZRNative "revoke-all-capabilities" revokeAll (funType tTyBool []) $
```

Comment:
As mentioned above let's ditch this and just have users use `(commit-tx)` etc.

Reply:
Fixed